system-variables.md

```diff
@@ -1,129 +1,836 @@
+
+- Scope: SESSION
+- Default value: 0
+- This variable is used to control whether to mark the corresponding SQL statement in the profile output to identify and troubleshoot performance issues.
+
+### tidb_record_plan_in_slow_log
+
+- Scope: SESSION
+- Default value: `1`
+- This variable is used to control whether to include the execution plan of slow queries in the slow log.
+
+### tidb_replica_read <span class="version-mark">New in v4.0</span>
+
+- Scope: SESSION
+- Default value: leader
+- This variable is used to control where TiDB reads data. Here are three options:
+    - leader: Read only from leader node
+    - follower: Read only from follower node
+    - leader-and-follower: Read from leader or follower node
+- See [follower reads](/follower-read.md) for additional details.
+
+### tidb_retry_limit
+
+- Scope: SESSION | GLOBAL
+- Default value: 10
+- This variable is used to set the maximum number of the retries. When a transaction encounters retryable errors (such as transaction conflicts, very slow transaction commit, or table schema changes), this transaction is re-executed according to this variable. Note that setting `tidb_retry_limit` to `0` disables the automatic retry.
+
+### tidb_row_format_version
+
+- Scope: GLOBAL
+- Default value: `2`
+- Controls the format version of the newly saved data in the table. In TiDB v4.0, the [new storage row format](https://github.com/pingcap/tidb/blob/master/docs/design/2018-07-19-row-format.md) version `2` is used by default to save new data.
+- If you upgrade from a TiDB version earlier than 4.0.0 to 4.0.0, the format version is not changed, and TiDB continues to use the old format of version `1` to write data to the table, which means that **only newly created clusters use the new data format by default**.
+- Note that modifying this variable does not affect the old data that has been saved, but applies the corresponding version format only to the newly written data after modifying this variable.
+
+### tidb_scatter_region
+
+- Scope: GLOBAL
+- Default value: 0
+- By default, Regions are split for a new table when it is being created in TiDB. After this variable is enabled, the newly split Regions are scattered immediately during the execution of the `CREATE TABLE` statement. This applies to the scenario where data need to be written in batches right after the tables are created in batches, because the newly split Regions can be scattered in TiKV beforehand and do not have to wait to be scheduled by PD. To ensure the continuous stability of writing data in batches, the `CREATE TABLE` statement returns success only after the Regions are successfully scattered. This makes the statement's execution time multiple times longer than that when you disable this variable.
+
+### tidb_skip_isolation_level_check
+
+- Scope: SESSION
+- Default value: 0
+- After this switch is enabled, if an isolation level unsupported by TiDB is assigned to `tx_isolation`, no error is reported.
```

Comment:
Is there any risk on turning on this variable? @cfzjywxk

Reply:
I have added a paragraph: "This helps improve compatibility with applications that set (but do not depend on) a different isolation level." I believe the history was a reporting app setting serializable.